tools/driver/cling.cpp

```diff
@@ -106,6 +106,17 @@ int main( int argc, char **argv ) {
     Interp.loadFile(Lib);
 
   cling::UserInterface Ui(Interp);
+
+  // Look for .cling_init in the home directory and if found process it like a cling script
+  llvm::SmallString<128> ClingInit;
+  llvm::sys::path::home_directory(ClingInit);
+
+  llvm::sys::path::append(ClingInit, ".cling_init");
+
+  if(llvm::sys::fs::exists(ClingInit)) {
+    Ui.getMetaProcessor()->readInputFromFile(ClingInit.c_str(), nullptr, -1, true);
```

Comment:
`readInputFromFile` takes a `StringRef`; better provide parameter names as comments for non-obvious argument values, better cast `-1` to `size_t`:
```suggestion
    Ui.getMetaProcessor()->readInputFromFile(ClingInit, /*result*/ nullptr, /*posOpenCurly*/ (size_t)-1, /*lineByLine*/ true);
```
Please repair my change to follow 80char limit.